crates/proof-of-sql/src/base/database/column.rs

```diff
@@ -422,8 +422,22 @@ impl ColumnType {
         )
     }
 
+    /// Returns the floor of the sqrt of the negative min integer
+    /// Returns `None` if the type is not a signed integer.
```

Comment:
You probably want to provide the formula as well as an example 
e.g. `f(NumericalType) := floor(sqrt(-NumericalType::MIN))`